.gitignore

```diff
@@ -1,2 +1,3 @@
-vendor
-composer.lock
+/vendor/
+/composer.lock
```

Comment:
Last thought for edge case: it would be terrible to see someone installing this as local dependency for its project to be bothered by conflicts when trying to require some other packages, because of our tilde-bounded constraints (e.g. `symfony/console:~3.1`). I hope the `application` type helps in that way 👼